sigllm/pipelines/prompter/mistral_prompter_0shot.json

```diff
@@ -0,0 +1,74 @@
+{
+    "primitives": [
+        "mlstars.custom.timeseries_preprocessing.time_segments_aggregate",
+        "sklearn.impute.SimpleImputer",
+        "sigllm.primitives.transformation.Float2Scalar",
+        "sigllm.primitives.prompting.timeseries_preprocessing.rolling_window_sequences",
+	    "sigllm.primitives.transformation.format_as_string",
+        
+        "sigllm.primitives.prompting.huggingface.HF",
+        "sigllm.primitives.transformation.parse_anomaly_response",
+        "sigllm.primitives.transformation.format_as_integer",
+        "sigllm/primitives/jsons/sigllm.primitives.prompting.anomalies.val2idx.json",
+        "sigllm/primitives/jsons/sigllm.primitives.prompting.anomalies.find_anomalies_in_windows.json",
```

Comment:
does it not work with `sigllm.primitives.prompting.anomalies.val2idx` and `sigllm.primitives.prompting.anomalies.find_anomalies_in_windows`?